jvm/src/main/scala/Main.scala

```diff
@@ -1,32 +1,36 @@
-import java.nio.file.{Files, Paths}
 import tastyquery.reader.TastyUnpickler
 import tastyquery.Contexts
-import tastyquery.api.ProjectReader
+import tastyquery.Contexts.{baseCtx, BaseContext}
+import tastyquery.reader.classfiles.Classpaths.Classpath
+import tastyquery.unsafe
 
-import java.net.URL
-import java.net.URLClassLoader
+import tastyquery.jdk.{ClassLoaders, ClasspathSources, IO, ProjectReaders}
 
 object Main {
+
   def main(args: Array[String]): Unit =
     args.toList match {
       case Nil =>
         // By default, read this project
-        val cl = ClassLoader.getSystemClassLoader
-        val classpath = cl.asInstanceOf[URLClassLoader].getURLs.map(_.getFile).toList
-        val reader = new ProjectReader
-        reader.read(classpath)
+        val classpath = ClasspathSources.classloaderURLs
+        ProjectReaders.read(classpath.toList)
       case "-cp" :: classpath =>
-        val reader = new ProjectReader
-        reader.read(classpath)
+        ProjectReaders.read(classpath)
       case "--standalone" :: filename :: Nil =>
-        val unpickler = new TastyUnpickler(Files.readAllBytes(Paths.get(filename)))
-        println(
-          unpickler.unpickle(new TastyUnpickler.TreeSectionUnpickler()).get.unpickle(using Contexts.empty(filename))
-        )
+        import unsafe.unlock.given unsafe.CanObserveIArray[Byte]
+        val unpickler = new TastyUnpickler(unsafe.asArray(IO.readAllBytes(filename)))
+        val ctx = Contexts.empty(Classpath.Empty, filename)
+        println(unpickler.unpickle(new TastyUnpickler.TreeSectionUnpickler()).get.unpickle(using ctx))
+      case "-usejavacp" :: Nil =>
```

Comment:
Is this really necessary? The availability of this option is already a historical liability in the compiler. Shouldn't we be able to always use `-cp` and explicitly provide the classpath instead?

Reply:
I haven't tested this main class at all, its just leftovers from prior work, I'd prefer to get rid of it for now